docs/components/SimpleInputDocs.js

```diff
@@ -25,14 +25,40 @@ class SimpleInputDocs extends React.Component {
         />
 
         <h3>Usage</h3>
+        <h5>baseColor <label>String</label></h5>
+        <p>The color of the <Code>input</Code> border on focus.</p>
```

Comment:
I would make a note that this prop is deprecated and to use `theme` instead. here's the warning we throw when someone uses that prop":https://github.com/mxenabled/mx-react-components/blob/master/src/utils/Deprecation.js